stryker.conf.json

```diff
@@ -0,0 +1,18 @@
+{
+  "files": [
+    "src/**/*.ts",
+    "test/**/*.ts"
+  ],
+  "mutate": [
+    "src/**/*.ts"
+  ],
+  "mutator": "typescript",
+  "reporters": [
+    "clear-text",
+    "progress"
+  ],
+  "testRunner": "jest",
+  "thresholds": {
+    "break": 100
```

Comment:
Explored whether to set `high` thinking that the 80% default wouldn't be matching. Doesn't change anything.